.github/workflows/test.yaml

```diff
@@ -8,5 +8,4 @@ jobs:
     uses: canonical/operator-workflows/.github/workflows/test.yaml@main
     secrets: inherit
     with:
-      self-hosted-runner: true
-      self-hosted-runner-label: "edge"
+      self-hosted-runner: false
```

Comment:
May I ask if there are special requirements that prevents us from using self-hosted runner please? :D

Reply:
Not a special requirement but I did this to reduce the load on our self-hosted runners as this workflow is quite lightweight :smile: